Source/ChocolateyGui.Shared/Models/Package.cs

```diff
@@ -5,71 +5,134 @@
 // --------------------------------------------------------------------------------------------------------------------
 
 using System;
+using System.Runtime.Serialization;
 
-namespace ChocolateyGui.Subprocess
-{
+namespace ChocolateyGui.Models
+{
+    [DataContract]
     public class Package
     {
+        [DataMember]
         public string[] Authors { get; set; }
 
+        [DataMember]
+
         public string Copyright { get; set; }
 
+        [DataMember]
+
         public string Dependencies { get; set; }
 
+        [DataMember]
+
         public string Description { get; set; }
 
+        [DataMember]
+
         public int DownloadCount { get; set; }
 
+        [DataMember]
+
         public string GalleryDetailsUrl { get; set; }
 
-        public string IconUrl { get; set; }
-
-        public string Id { get; set; }
-
-        public bool IsAbsoluteLatestVersion { get; set; }
-
+        [DataMember]
+
+        public string IconUrl { get; set; }
+
+        [DataMember]
+
+        public string Id { get; set; }
+
+        [DataMember]
+
+        public bool IsAbsoluteLatestVersion { get; set; }
+
+        [DataMember]
+
         public bool IsInstalled { get; set; }
 
+        [DataMember]
+
```

Comment:
looks like extra lines inserted here. Issues with flipping this away or to UTF8 usually causes this.